products/BellHybrid/apps/application-bell-background-sounds/windows/BGSoundsTimerSelectWindow.cpp

```diff
@@ -115,7 +117,10 @@ namespace gui
     }
     void BGSoundsTimerSelectWindow::updateBottomDescription()
     {
-        const bool isDescriptionVisible = UTF8ToTimerValue(spinner->getCurrentValue()) != offValue;
+        const auto currentVal = spinner->getCurrentValue();
+        bottomDescription->setText(utils::language::getCorrectMinutesNumeralForm(UTF8ToTimerValue(currentVal).count()));
```

Comment:
what is that ?

Reply:
bottom text contains the word "minutes" in English. In Polish we need to conjugate so it will be 15 - "minut", 23 - "minuty " etc. Thats what `utils::language::getCorrectMinutesNumeralForm` does.

The rest is just conversion of minutes format (spinner has string string, while the mentioned method requires an uint).

Is that what you meant?

Reply:
ok :)